scripts/example_bundle_run.py

```diff
@@ -34,14 +34,14 @@
 with open('/mnt/scripts/instcat_worklist_a.json') as fp:
     worklist_a = json.load(fp)
 
-# We need to define the max number of threads and max number of instances per node for this. We choose
-# the fiducial values from Theta. Note that the max_instances_node is a non-trivial calculation.
-# We assume 10 GB memory per imSim call + 1 GB per thread + 5 GB floating for sharp increases.
-# This may vary depending on your version of imSim and architecture.
+# we now define key characteristics of the imSim version + architecture.
+mem_per_thread = 2      # the amount of memory a given sensor thread is expected to max out at
+mem_per_instance = 10   # the amount of shared memory an imSim container is expected to have
 
-max_threads_node = 64
-max_instances_node = 10
-bundle_list_a = jbu.determine_bundles(worklist_a, max_threads_node, max_instances_node)
+mem_per_node = 96-5     # the available memory on a compute node to target for use, minus some leeway
+threads_per_node = 68*4 # the number of available threads to be used on a given node
```

Comment:
so `threads_per_node` would normally be number of cores on the node?

Reply:
A potential difference is that you may want to try to take advantage of hyperthreading; in the case of Cori the memory limitations means this never actually comes up, but for Theta you have 64 cores with 4 threads each... and enough memory that conceivably you could run around ~80 processes at once if you want to take advantage of hyperthreading.